### PR TITLE
Fix mag declination css for dark mode

### PR DESCRIPTION
**Pull request by haslinghuis**
Element did not work in dark mode. Also aligned it with other elements.

BEFORE:

image

AFTER:

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Comment by chmelevskij** [comment]
I love the size of these PRs ❤️ makes review. so much easier

**Pull request merged**
